VirtoCommerce.Platform.Web/Controllers/Api/AssetsController.cs

```diff
@@ -113,6 +118,12 @@ public AssetsController(IBlobStorageProvider blobProvider, IBlobUrlResolver urlR
                         RelativeUrl = fileUrl,
                         Url = _urlResolver.GetAbsoluteUrl(fileUrl)
                     });
+
```

Comment:
It is necessary to eliminate duplication of code and prevent the creation of duplicate assets when an asset with the same Url and tenants is uploaded

Reply:
What about this comment?